File: src/main/scala/exoskeleton/common/ExoTiles.scala
package exoskeleton.common

import cpw.mods.fml.common.registry.GameRegistry
import exoskeleton.common.tile.{TileEntityAssembler, TileEntityModifier, TileEntityToolbox}

object ExoTiles{
  def init(): Unit ={
    GameRegistry.registerTileEntity(classOf[TileEntityToolbox], "tileToolbox");
    GameRegistry.registerTileEntity(classOf[TileEntityAssembler], "tileEntityAssembler");
    GameRegistry.registerTileEntity(classOf[TileEntityModifier], "tileEntityModifier");
  }
}

File: src/main/scala/exoskeleton/common/EXOTiles.java
package exoskeleton.common;

import cpw.mods.fml.common.registry.GameRegistry;
import exoskeleton.common.tile.TileEntityAssembler;
import exoskeleton.common.tile.TileEntityModifier;
import exoskeleton.common.tile.TileEntityToolbox;

public final class EXOTiles{
    public static void initialize(){
        GameRegistry.registerTileEntity(TileEntityToolbox.class, "tileEntityToolbox");
        GameRegistry.registerTileEntity(TileEntityAssembler.class, "tileEntityAssembler");
        GameRegistry.registerTileEntity(TileEntityModifier.class, "tileEntityModififer");
    }
}